### ISSUE TITLE
Git clone  git clone git@gitlab.sz.sensetime.com:open-mmlab/mmpose.git doesn't work

### ISSUE THREAD

**Issue by akileshrajan** [open-mmlab/mmpose#90]
Don't know if it is just me. Cloning the repository form the mentioned gitlab link doesn't work. The connection times out.

```
(HRNet) akilesh@heracleiapc:~/PyCharmProjects$ git clone git@gitlab.sz.sensetime.com:open-mmlab/mmpose.git
Cloning into 'mmpose'...
ssh: connect to host gitlab.sz.sensetime.com port 22: Connection timed out
fatal: Could not read from remote repository.

Please make sure you have the correct access rightsand the repository exists.
```

I cloned with the github link though.!

**Comment by jin-s13** [comment]
Thank you for reporting this issue.
Yes, we should use the GitHub link. I will update the Readme and clarify this.
Thanks again.

**Comment by jin-s13** [comment]
Will be fixed in #91
